lib/src/Uncertainty/Model/openturns/DistributionImplementation.hxx

```diff
@@ -1687,6 +1687,74 @@ protected:
     const DistributionImplementation * p_distribution_;
   };  // class EntropyKernel
 
+  // Class used to wrap the computePDF()**2  method for interpolation purpose
```

Comment:
"interpolation" should be replaced with "integration"

Reply:
Done